managedtenants/data/selectorsyncset.yaml.j2

```
@@ -29,7 +29,7 @@ items:
       - apiVersion: v1
         kind: Secret
         metadata:
-          name: {{secret['name']}}
+          name: {{secret['destinationSecretName'] | default(secret['name'])}}
```

```suggestion
          name: {{secret.get('destinationSecretName') or secret['name']}}
```
shouldn't it be this? 

Considering the Pythonic nature of Jinja, I am afraid that `secret['destinationSecretName']` might raise KeyError instead of returning `None` just the way Python does?

Would love to be corrected in case you already tested this :)

---

Yup, I already tested this. When `destinationSecretName` isn't defined, it just fallbacks to the default value, i.e. `secret['name']`. I can change it as per your suggestion too. What do you suggest?